<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selection_help</class>
 <widget class="QWidget" name="selection_help">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>282</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Allowed Head Loss / 
Static Head Difference</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Approx / Known 
Internal Diameter</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>k-factor in Pipe 
or for Valve</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Flow through
the Element</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Non-Linearity
for valve (n)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="h_le">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="D_le">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="k_le">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="q_le">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="n_le">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Roughness of Pipe</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="e_le">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Length of Pipe</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="L_le">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="labelAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="formAlignment">
     <set>Qt::AlignCenter</set>
    </property>
    <item row="0" column="0" colspan="2">
     <widget class="QRadioButton" name="D_rb">
      <property name="text">
       <string>Ca&amp;lculate Internal Diameter
for Pipe Segment</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QRadioButton" name="Hd_rb">
      <property name="text">
       <string>Calculate Re&amp;quired
Head of Booster</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QRadioButton" name="o_rb">
      <property name="text">
       <string>Calculate Required
Opening of &amp;Valve </string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Go_pb">
      <property name="minimumSize">
       <size>
        <width>132</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Go</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>99</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="Result_le">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="result_lb">
      <property name="text">
       <string>Result</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>h_le</tabstop>
  <tabstop>D_le</tabstop>
  <tabstop>e_le</tabstop>
  <tabstop>L_le</tabstop>
  <tabstop>k_le</tabstop>
  <tabstop>q_le</tabstop>
  <tabstop>n_le</tabstop>
  <tabstop>D_rb</tabstop>
  <tabstop>Hd_rb</tabstop>
  <tabstop>o_rb</tabstop>
  <tabstop>Go_pb</tabstop>
  <tabstop>Result_le</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
